sql/pgwire/server.go

```diff
@@ -204,7 +204,7 @@ func (s *Server) ServeConn(conn net.Conn) error {
 		v3conn := makeV3Conn(conn, s.executor, s.metrics, sessionArgs)
 		defer v3conn.finish()
 		if argsErr != nil {
-			return v3conn.sendInternalError(err.Error())
+			return v3conn.sendInternalError(argsErr.Error())
```

Comment:
I assume the reasoning was that we're not checking the error immediately (see the comment above), so the error deserves a more descriptive name. I think I agree with that logic here, as using `err` outside of its usual `if _, err := ...; err != nil {` pattern can get pretty confusing (not to mention the potential bugs introduced by unintended future shadowing if the declaration and usage are split).